.github/pull_request_template.md

```diff
@@ -21,6 +21,19 @@ Describe the tests that you ran to verify your changes.
 
 Make sure to provide instructions for the maintainer as well as any relevant configurations.
 
+- [ ] Basic tests (linting, compilation, formatting, unit/integration tests)
+- [ ] Patch-specific tests (correctness, functionality coverage)
+
+### Infrastructure QA
+
+For any unchecked item, authors must provide a justification that describes why the test was not necessary.
+
```

Comment:
This can be removed: Intended for internal guidelines.

Reply:
Fixed by https://github.com/iotaledger/iota/pull/6368/commits/f6a88177fdcca050907923ffd31ed62030166885